### PR TITLE
Add osu!idol 2020 auditions announcement

### PR DESCRIPTION
**Pull request by Flanster**
osu!Idol 2020 will begin shortly so we'd like to announce it with this newspost.

### CODE REVIEW

**Comment by Ephemeralis** [comment]
have we linked directly from a newspost to a discord server before? i think it might be preferable to make a forum thread for osu!idol as well and have the server details in there versus linking it straight

**Comment by Flanster** [comment]
There isn't a plan for a thread that's why there's only a Discord linked. Music Hall's traffic is low in itself so it doesn't serve much purpose other than needing to post the same info twice. This is to save time, effort and direct people to the proper place where things happen and that isn't threads anymore.

**Comment by Ephemeralis** [comment]
i suppose this is fine then, though for future contests it'd be nice to have a forum endpoint for it as well

File: news/2020-09-01-osu-idol-2020-auditions-now-open.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+You bet. Once the community voting has concluded, the finalists will be receiving these goodies as followed:
+
+- **1st:** 2020 Winner badge for their osu! profile, a special seperate 2020 Winner role in our discord server, 6 months of osu! supporter and a chance to be a judge in the next idol contest.
+- **2nd:** 6 months of osu! supporter and a chance to be a judge in the next idol contest.
+- **3rd:** 3 months of osu! supporter and a chance to be a judge in the next idol contest.
+
+### Where do I go?
+
+Please join our [osu!idol discord server](https://discord.gg/2xfgKaU) hub. There you will can find the requirements, contest rules, submission sheet, FAQ, ask questions, and talk to the community!
```
Comment (TPGPL) [inline news/2020-09-01-osu-idol-2020-auditions-now-open.md:27]:
```suggestion
Please join our [osu!idol discord server](https://discord.gg/2xfgKaU) hub. There you can find the requirements, contest rules, submission sheet, FAQ, ask questions, and talk to the community!
```

File: news/2020-09-01-osu-idol-2020-auditions-now-open.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "osu!idol 2020 Auditions Now Open!"
+date: 2020-09-01 17:00:00 +0000
+---
+
+Remember that time where people were singing and our ears were ringing with joy? It's happening again. If you are interested in singing and competing to becoming osu!'s the next idol you should grab a mic and join us on our singing adventure!
```
Comment (TPGPL) [inline news/2020-09-01-osu-idol-2020-auditions-now-open.md:7]:
```suggestion
Remember that time when people were singing and our ears were ringing with joy? It's happening once again! If you are interested in singing and competing to become the osu!'s next idol, you should grab a mic and join us on our singing adventure!
```

File: news/2020-09-01-osu-idol-2020-auditions-now-open.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+![](/wiki/shared/news/2020-09-01-osu-idol-2020-auditions-now-open/2020.jpg)
+
+osu!idol is the community driven singing contest where contestants participate as a solo or duet, in multiple stages consisting of various genres, showcasing their vocal skills to our panel of judges. They will evaluate their performances and determine who makes it through the stages, eventually ending up in the finals. Voting for the winner will be conducted by you, the community, to decide who will be the next osu!idol!
```
Comment (TPGPL) [inline news/2020-09-01-osu-idol-2020-auditions-now-open.md:11]:
```suggestion
osu!idol is the community driven singing contest where contestants participate as a solo or duet, in multiple stages consisting of various genres, showcasing their vocal skills to our panel of judges. The judges will evaluate their performances and determine who makes it through the stages, eventually ending up in the finals. Voting for the winner will be conducted by you, the community, to decide who will be the next osu!idol!
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

File: news/2020-09-01-osu-idol-2020-auditions-now-open.md (hunk 1 above)
Comment (Flanster) [reply news/2020-09-01-osu-idol-2020-auditions-now-open.md:27]: nice typo

File: news/2020-09-01-osu-idol-2020-auditions-now-open.md (hunk 2 above)
Comment (Flanster) [reply news/2020-09-01-osu-idol-2020-auditions-now-open.md:7]: That sounds much better.

File: news/2020-09-01-osu-idol-2020-auditions-now-open.md (hunk 3 above)
Comment (Flanster) [reply news/2020-09-01-osu-idol-2020-auditions-now-open.md:11]: Using "the judges" right after saying "judges" sounds too rapidly repetitive

Comment (zeusminus) [reply news/2020-09-01-osu-idol-2020-auditions-now-open.md:11]: "**They** will evaluate **their** performances..." sounds a bit odd to me, plus *they* pronoun here can refer to *the contestants* given the previous sentence. As TPGPL suggested, *The judges* would be the safest and the least ambiguous phrasing here, from what I see.

Comment (Flanster) [reply news/2020-09-01-osu-idol-2020-auditions-now-open.md:11]: Fair point, though that still doesnt sound well to me. Perhaps this part needs to be worded better altogether.

_"osu!idol is the community driven singing contest where contestants showcase their vocal skills as a solo or duet, in multiple stages consisting of various genres. Our panel of judges will evaluate their performances and determine who makes it through the stages, eventually ending up in the finals. Voting for the winner will be conducted by you, the community, to decide who will be the next osu!idol!_

Comment (TPGPL) [reply news/2020-09-01-osu-idol-2020-auditions-now-open.md:11]: yeah i think that `They` in this context refers more to contestants than judges, so the sentence might be misleading for readers

Comment (TPGPL) [reply news/2020-09-01-osu-idol-2020-auditions-now-open.md:11]:
```suggestion
osu!idol is the community driven singing contest where contestants participate as a solo or duet, in multiple stages consisting of various genres, showcasing their vocal skills to our jury. The judges will evaluate their performances and determine who makes it through the stages, eventually ending up in the finals. Voting for the winner will be conducted by you, the community, to decide who will be the next osu!idol!
```
if you dont want to repeat the `judges` word, then this suggestion might be better?

Comment (Flanster) [reply news/2020-09-01-osu-idol-2020-auditions-now-open.md:11]: Read my suggestion in italic above @TPGPL

Comment (TPGPL) [reply news/2020-09-01-osu-idol-2020-auditions-now-open.md:11]: looks fine

Comment (Flanster) [reply news/2020-09-01-osu-idol-2020-auditions-now-open.md:11]: Will apply that then.

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**